src/User/Handlers.hs

```diff
@@ -22,5 +24,18 @@ handleUserAdd = do
   sd <- liftIO getSystemDay
   (view, muf) <- runForm "user_form" $ userForm sd
   case muf of
-    Just uf -> insertUser uf >> redirect "/users"
-    Nothing -> heistLocal (bindDigestiveSplices view) $ render "users_add" 
\ No newline at end of file
+    Just uf -> do
+      r <- insertUser uf
```

Comment:
@kinorius123 Could you please rename this to something like `numCreatedUsers`? We do prefer explicit names.